src/sagemaker/estimator.py

```diff
@@ -3429,6 +3449,12 @@ def _distribution_configuration(self, distribution):
                     "dataparallel"
                 ].get("custom_mpi_options", "")
 
+        if "multi_worker_mirrored_strategy" in distribution:
+            mwms_enabled = distribution.get("multi_worker_mirrored_strategy").get("enabled", False)
+            if mwms_enabled:
+                self._validate_mwms_config()
```

Comment:
Please also add checks to make sure this option is not enabled together with other incompatible options.

Reply:
Done